scripts/events_module/short/short_event.py

```diff
@@ -1,20 +1,34 @@
 from typing import List
 
+from scripts.cat.personality import Personality
+from scripts.cat.skills import SkillPath
+from scripts.game_structure import constants
+
 
 class ShortEvent:
     """
     A moon event that only affects the moon it was triggered on.  Can involve two cats directly and be restricted by various constraints.
     - full documentation available on GitHub wiki
     """
 
+    num_of_traits = len(Personality.trait_ranges["normal_traits"].keys()) + len(
+        Personality.trait_ranges["kit_traits"].keys()
+    )
+    num_of_skills = len([*SkillPath])
```

Comment:
```suggestion
    num_of_skills = len(SkillPath)
```
Is there a reason this doesn't work? /genq